google/cloud/forseti/services/inventory/base/cloudasset.py

```diff
@@ -133,17 +127,43 @@ def load_cloudasset_data(engine, config):
                                                config,
                                                root_id,
                                                content_type))
+        gcs_paths = []
 
         for future in concurrent.futures.as_completed(futures):
-            gcs_object = future.result()
-            try:
-                assets = _stream_gcs_to_database(gcs_object,
-                                                 engine,
-                                                 storage_client)
-                imported_assets += assets
-            except StreamError as e:
-                LOGGER.error('Error streaming data from GCS to Database: %s', e)
-                return _clear_cai_data(engine)
+            gcs_paths.append(future.result())
+        return gcs_paths
```

Comment:
Make this yield the gcs path instead of return a list, then when the first file is complete the download can begin, rather than having to wait for both exports to complete.

Reply:
updated to using yield